acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/site/ps/SearchPage.java

```diff
@@ -0,0 +1,55 @@
+package uk.nhs.digital.ps.test.acceptance.pages.site.ps;
```

Comment:
move to "common" package instead of "ps"

Reply:
Good idea